lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -47,6 +47,34 @@
   }
```

Comment:
The mockup shows red-40 for the error messages, but the impl has red-45, which is what we have as our standard error message color all across the implementation.  Is the mockup wrong or do we have the error color wrong everywhere in the impl?

Reply:
> The mockup shows red-40 for the error messages, but the impl has red-45, which is what we have as our standard error message color all across the implementation. Is the mockup wrong or do we have the error color wrong everywhere in the impl?

I checked with Jason on this- across Kaleido form validation is set as red-40, which we should be using (since it has already been tested for accessibility contrast). I looked into the `--nx-color-validation-valid` which is also off (Kaleido is green-25, RSC is green-30). I'll bring this up tomorrow at WW so we can agree and make a ticket to resolve this (since I assume it's out of scope here as it will be changing a lot of snapshots?)